ironfish/src/migrations/data/013-wallet-2.ts

```diff
@@ -242,17 +255,69 @@ export class Migration013 extends Migration {
     return accounts
   }
 
+  async migrateTransactions(
+    transactionsStoreOld: Stores['old']['transactions'],
+    transactionsStoreNew: Stores['new']['transactions'],
+    headersStoreOld: Stores['old']['headers'],
+    tx: IDatabaseTransaction,
+    logger: Logger,
+  ): Promise<BufferMap<true>> {
+    const dropped = new BufferMap<true>()
+
+    let countMigrated = 0
+    let countDropped = 0
+
+    for await (const [transactionHash, transactionValue] of transactionsStoreOld.getAllIter(
+      tx,
+    )) {
+      const transactionHashHex = transactionHash.toString('hex')
+      await transactionsStoreOld.del(transactionHash, tx)
+
+      const migrated: DatabaseStoreValue<typeof transactionsStoreNew> = {
+        ...transactionValue,
+        sequence: null,
+      }
+
+      if (transactionValue.blockHash) {
+        const blockHash = Buffer.from(transactionValue.blockHash, 'hex')
+        const header = await headersStoreOld.get(blockHash)
+
+        if (header) {
+          migrated.sequence = header.header.sequence
+          countMigrated++
+        } else {
+          logger.debug(
+            `\tDropping TX ${transactionHashHex}: block not found ${transactionValue.blockHash}`,
+          )
+          await transactionsStoreOld.del(transactionHash, tx)
```

Comment:
Doesn't look like this is needed -- all transactions get deleted from `transactionsStoreOld` below